### PR TITLE
deprecates PID policy

### LINKED ISSUE #1487: Autoscaling docs are not up-to-date
https://paasta.readthedocs.io/en/latest/autoscaling.html doesn't talk of proportional decision_policy, forecast_policy, moving_average_window_seconds, good_enough_window etc.

### PR DESCRIPTION
**Pull request by fboxwala**
Fixes #1487 and PAASTA-13337

### CODE REVIEW

File: docs/source/autoscaling.rst [hunk 1]
```diff
@@ -106,6 +106,19 @@ Decision policies
 
 The currently available decicion policies are:
 
+:proportional:
```
Comment (solarkennedy) [inline docs/source/autoscaling.rst]: Can you explicitly say what the defaults are for the extra params below?

File: docs/source/autoscaling.rst [hunk 2]
```diff
@@ -117,6 +130,22 @@ The currently available decicion policies are:
...
+
+:current:
+  Assumes current load will remain the same as the current value for the near future.
+
+:moving_average:
+  Assumes total load will remain near the average of data points within a window.
+
+  Extra parameters:
+   :moving_average_window_seconds:
+     The number of seconds to load data points over in order to calculate the average.
```
Comment (solarkennedy) [inline docs/source/autoscaling.rst]: default?

**Comment by fboxwala** [comment]
Docs updates moved to #1585

**Comment by mjksmith** [comment]
Rest in peace PID autoscaler.

To live on in the hearts of others is to never die.

**Pull request merged**